### PR TITLE
WIP: Prepare for openshift testing

### PR DESCRIPTION
**Pull request by pvalena**
Add run-openshift and symlink to lib.

### CODE REVIEW

**Comment by omron93** [comment]
[test]

**Comment by pvalena** [comment]
@omron93, I think this commit itself should not break anything- appart from updating 'common' folder, which should be fixed anyway :)

@hhorak, I would you prefer merging this and subsequently adding the jenkins job (if needed) to run the tests and then creating the tests themselves, WDYT?

**Comment by pvalena** [comment]
From what I figured, the CI just hung:

```
06:57:30 Step 8 : RUN yum repolist > /dev/null &&     yum install -y yum-utils &&     yum-config-manager --disable \* &> /dev/null &&     yum-config-manager --enable rhel-server-rhscl-7-rpms &&     yum-config-manager --enable rhel-7-server-rpms &&     yum-config-manager --enable rhel-7-server-optional-rpms &&     INSTALL_PKGS="nss_wrapper bind-utils gettext hostname rh-nginx18 rh-nginx18-nginx" &&     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all
06:57:30  ---> Running in b4227553f6ef
08:57:12 Killed by signal 15.
```

**Comment by pvalena** [comment]
[test]

**Comment by pvalena** [comment]
Please do not merge until the test is edited.

File: test/run-openshift [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+#
+# Test the ???? image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+#############################################
+#                                           #
+#    THIS IS A COPY FROM SCLORG/MARIADB     #
+#         AND NEEDS TO BE EDITED            #
```
Comment (pvalena) [inline test/run-openshift]: => WIP <=

**Comment by omron93** [comment]
> From what I figured, the CI just hung:

There is 2h timeout for tests.

**Comment by hhorak** [comment]
This is already obsolete by #40.

**Pull request closed without merging**